src/style.css

```diff
@@ -0,0 +1,154 @@
+* {
+  margin: 0;
+}
+
+body {
+  margin: 0;
```

Comment:
Why that property ? You've already set `margin: 0;` up.

Reply:
Thank you for pointing this out!